src/aero/actuator/ActuatorExecutorsFASTNgp.C

```diff
@@ -25,33 +25,46 @@ ActuatorLineFastNGP::ActuatorLineFastNGP(
 void
 ActuatorLineFastNGP::operator()()
 {
-  actBulk_.zero_source_terms(stkBulk_);
+  //Zero the (body-force) actuator source term 
+  actBulk_.zero_source_terms(stkBulk_); 
 
-  // set range policy to only operating over points owned by local fast turbine
+  //Range for Kokkos parallel-for -- set range policy to only operating over points owned by local fast turbine
   auto fastRangePolicy = actBulk_.local_range_policy();
 
+  //Interpolate velocity to actuator points. This happens before fine search? Is this from previous timestep
   RunInterpActuatorVel(actBulk_, stkBulk_);
 
+  // Add FLLC correction to velocity field. Is this term treated explicitly? Also seems like it’s from previous step
```

Comment:
Yes it is explicit and there is a lag.  The value has a heavy relaxation factor.